deployment/submit_emr_job_train.py

```diff
@@ -23,57 +23,56 @@
 from time import gmtime, strftime
 import daiquiri
 
-import recommendation_engine.config.cloud_constants as config
+# import recommendation_engine.config.cloud_constants as config
```

Comment:
If you don't need this here best to remove the line than comment it.

Reply:
Ok, I will remove it all the commented lines.